runtimes/protocol/identity-management.ts

```diff
@@ -0,0 +1,22 @@
+import { ProtocolRequestType } from './lsp'
+
+export type ListProfilesParams = {}
+export type ListProfilesResult = {
+    profiles?: any[]
+    ssoSessions?: any[]
+}
+export type ListProfilesError = {}
```

Comment:
I'm working through a concept of handling enriched errors now.  I'll submit code/PR to the team in a doc to discuss further.  For now, an empty error type is fine.